public/app/features/alerting/unified/components/notification-policies/NotificationPoliciesList.tsx

```diff
@@ -218,34 +220,31 @@ export const NotificationPoliciesList = () => {
       )}
       {hasPoliciesData && (
         <Stack direction="column" gap={1}>
-          {rootRoute && (
-            <NotificationPoliciesFilter
-              onChangeMatchers={setLabelMatchersFilter}
-              onChangeReceiver={setContactPointFilter}
-              matchingCount={routesMatchingFilters.matchedRoutesWithPath.size}
-            />
-          )}
-          {rootRoute && (
-            <Policy
-              receivers={receivers}
-              currentRoute={rootRoute}
-              contactPointsState={contactPointsState.receivers}
-              readOnly={!hasConfigurationAPI}
-              provisioned={rootRoute[ROUTES_META_SYMBOL]?.provisioned}
-              alertManagerSourceName={selectedAlertmanager}
-              onAddPolicy={openAddModal}
-              onEditPolicy={openEditModal}
-              onDeletePolicy={openDeleteModal}
-              onShowAlertInstances={showAlertGroupsModal}
-              routesMatchingFilters={routesMatchingFilters}
-              matchingInstancesPreview={{
-                groupsMap: routeAlertGroupsMap,
-                enabled: Boolean(canSeeAlertGroups && !instancesPreviewError),
-              }}
-              isAutoGenerated={false}
-              isDefaultPolicy
-            />
-          )}
+          <NotificationPoliciesFilter
+            onChangeMatchers={setLabelMatchersFilter}
+            onChangeReceiver={setContactPointFilter}
+            matchingCount={routesMatchingFilters.matchedRoutesWithPath.size}
+          />
+          <Policy
+            receivers={receivers}
+            // add the timing defaults to the default policy to make sure child policies inherit properly
+            currentRoute={defaults(rootRoute, TIMING_OPTIONS_DEFAULTS)}
```

Comment:
The _actual_ fix. The default policy should have the default timing options included in its definition.

This _does_ mean that a user can never choose to save a default policy with either of the timing options set to `undefined` or otherwise leave that empty.